Packs/XMCyber/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### XM Cyber
+- **Breaking Change** ip and xmcyber-hostname commands changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
```suggestion
- **Breaking Change** the ***ip*** and ***xmcyber-hostname*** commands were changed to return multiple entries (entry per indicator) instead of a single entry.
```